lasers:
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 0, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.4363323129985824,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 1, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.07330382858376185, vert_correction: -0.017453292519943295,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 2, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.029094638630745476,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 3, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.2729520417193932,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 4, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.19739673840055869,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 5, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 6, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.07330382858376185, vert_correction: -0.011641346110802179,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 7, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.15433946575385857,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 8, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.12660618393966866,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 9, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.07330382858376185, vert_correction: 0.005811946409141118,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 10, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.005811946409141118,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 11, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.10730284241261137,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 12, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.07330382858376185, vert_correction: -0.0930784090088576,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 13, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: 0.02326523892908441,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 14, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.07330382858376185, vert_correction: 0.011641346110802179,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 15, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.06981317007977318,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 16, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.08145451619057535,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 17, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.07330382858376185, vert_correction: 0.029094638630745476,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 18, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: 0.017453292519943295,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 19, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.07330382858376185, vert_correction: -0.06400122367063206,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 20, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.07330382858376185, vert_correction: -0.058171823968971005,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 21, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: 0.058171823968971005,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 22, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: 0.04071853144902771,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 23, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.04654793115068877,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 24, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: -0.05235987755982989,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 25, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: 0.12217304763960307,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 26, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: 0.08145451619057535,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 27, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.07330382858376185, vert_correction: -0.04071853144902771,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 28, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.07330382858376185, vert_correction: -0.03490658503988659,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 29, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: 0.2617993877991494,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 30, max_intensity: 255,
  min_intensity: 0, rot_correction: -0.024434609527920613, vert_correction: 0.18034487160857407,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 31, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.024434609527920613, vert_correction: -0.02326523892908441,
  vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 32, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 33, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 34, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 35, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 36, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 37, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 38, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 39, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 40, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 41, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 42, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 43, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 44, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 45, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 46, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 47, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 48, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 49, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 50, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 51, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 52, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 53, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 54, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 55, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 56, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 57, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 58, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 59, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 60, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 61, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 62, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 63, max_intensity: 255,
  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
num_lasers: 64
